andela/ah-frontend-valkyrie
# 162633464: Users should be able to create articles

**What does this PR do?**

This PR allows users to create, read, update and delete articles.

**Description of Task completed?**
* Create new article
* Retrieve all articles
* View all articles
* Update an article
* Delete an article

**How should this be manually tested?**

- Run `npm install` && `npm start` to run the application.
- Check your redux dev tools to confirm that the state in your app appears in the dev tools.

**Any background context you want to provide?**

- For redux to work with react, remember to install react-redux by running `npm install react-redux`.

**What are the relevant pivotal tracker stories?**

[DELIVERS #162633464](https://www.pivotaltracker.com/n/projects/2231917/stories/162633464)
